Tools/LyTestTools/ly_test_tools/o3de/editor_test_utils.py

```diff
@@ -28,7 +29,7 @@ def compile_test_case_name_from_request(request):
         callspec = request.node.callspec.id
         compiled_test_case_name = f"{test_case_name}[{callspec}]"
     except Exception as e:
-        logging.warning(f"Error reading test case name for TIAF. {e}")
+        logging.warning(f"Error reading test case name in LyTestTools for TIAF. {exceptions.LyTestToolsException(e)}")
```

Comment:
Shouldn't wrap an exception that is only reported like this. Wrapping should be done when (re)raising